lib/std/os/uefi/protocol/managed_network.zig

```diff
@@ -8,58 +8,209 @@ const Time = uefi.Time;
 const SimpleNetwork = uefi.protocol.SimpleNetwork;
 const MacAddress = uefi.MacAddress;
 const cc = uefi.cc;
+const Error = Status.Error;
 
 pub const ManagedNetwork = extern struct {
     _get_mode_data: *const fn (*const ManagedNetwork, ?*Config, ?*SimpleNetwork) callconv(cc) Status,
-    _configure: *const fn (*const ManagedNetwork, ?*const Config) callconv(cc) Status,
-    _mcast_ip_to_mac: *const fn (*const ManagedNetwork, bool, *const anyopaque, *MacAddress) callconv(cc) Status,
-    _groups: *const fn (*const ManagedNetwork, bool, ?*const MacAddress) callconv(cc) Status,
-    _transmit: *const fn (*const ManagedNetwork, *const CompletionToken) callconv(cc) Status,
-    _receive: *const fn (*const ManagedNetwork, *const CompletionToken) callconv(cc) Status,
-    _cancel: *const fn (*const ManagedNetwork, ?*const CompletionToken) callconv(cc) Status,
-    _poll: *const fn (*const ManagedNetwork) callconv(cc) Status,
+    _configure: *const fn (*ManagedNetwork, ?*const Config) callconv(cc) Status,
+    _mcast_ip_to_mac: *const fn (*ManagedNetwork, bool, *const anyopaque, *MacAddress) callconv(cc) Status,
+    _groups: *const fn (*ManagedNetwork, bool, ?*const MacAddress) callconv(cc) Status,
+    _transmit: *const fn (*ManagedNetwork, *CompletionToken) callconv(cc) Status,
+    _receive: *const fn (*ManagedNetwork, *CompletionToken) callconv(cc) Status,
+    _cancel: *const fn (*ManagedNetwork, ?*const CompletionToken) callconv(cc) Status,
+    _poll: *const fn (*ManagedNetwork) callconv(cc) Status,
+
+    // TODO: according to the spec, "Other" errors are also possible for some of these fns,
+    // but i think it's referring to warnings?
```

Comment:
I don't think they refer to warnings, this seems to be dependent on the "MNP child driver" (whatever that is) - so we'll have to expect any UEFI error here.